docs/pywbemcligeneraloptions.rst

```diff
@@ -21,15 +21,13 @@ information on each command-group arguments and options):
   * Port: (optional) defines WBEM server port to be used [Defaults: 5988(HTTP)
     and 5989(HTTPS)]. (EnvVar: PYWBEMCLI_SERVER).
```

Comment:
The mentioning of env var PYWBEMCLI_SERVER is part of the list item for port. However, it applies to the entire `--server` option. Move out of the list item.

Reply:
DONE